begin/cmdline.py

```diff
@@ -163,7 +169,8 @@ def create_parser(func, env_prefix=None, config_file=None, config_section=None,
     arguments will raise a ValueError exception. A prefix on expected
     environment variables can be added using the env_prefix argument.
     """
-    defaults = DefaultsManager(env_prefix, config_file, func.__name__)
+    section = config_section if config_section is not None else func.__name__
```

Comment:
That's the single point I could find which leads to setting self._section to the value passed via config_section in the begin.start() call. Otherwise self._section will always be set to func.__name__ no matter what is passed in begin.start(). Am I missing something?